### DISCUSSION TITLE
Lost admin password

### DISCUSSION THREAD

**Discussion by demuxer86** [the-djmaze/snappymail#848] [General]
Setup is Proxmox > Truenas > Nextcloud (truecharts) > Snappymail. 

After initial load, I did not write down the temporary password because I have mental health issues.   How do I reset without blowing up my entire nextcloud install?   I have truetool installed, but can't find application.ini... I saw elsewhere that commenting out the password field in application.ini will reset the password, but I can't find that anywhere in the containers.

**Comment by cm-schl** [comment]
Hi,
normally there is no need (anymore) to write down the admin password as SnappyMail automatically logs you in to the admin panel if you are logged in as Nextcloud administrator.
As Nextcloud admin you can enter into the Nextcloud settings, go to additional settings and should see a link to enter into the admin panel.

**Reply by demuxer86** [reply]
Weird... I just looked at it again and now it gave me a temp password... that worked.   I guess that's that.   Thanks for the reply!

Now I need to figure out how to migrate my office 365 email to Namecheap so I can use these IMAP apps.  I am done with microsoft and google.   Their security has risen to the level of "use our web apps and nothing else".   Screw that!
